gcovr/writer/json.py

```diff
@@ -58,6 +57,26 @@
 PRETTY_JSON_INDENT = 4
 
 
+get_filename = None
```

Comment:
Not a fan of global variables. The code can instead call a function like `prefixed_filename(filename, options)`.

Reply:
I wanted to save the time for normalizing the value of json_base but this can also be done in the commandlin parser.